### PR TITLE
BaSP-MR-147: Private routes and navigation bar for different entities

### PR DESCRIPTION
**Pull request by josedaniele**
The ADMIN entity now has access to the "Classes" , enabling them to manage and oversee class-related functionalities .
They can navigate through the "Activities" section, allowing them to create, edit, and see activities .
Access to the "Subscriptions" .
ADMIN users can view the comprehensive "Members" list, enabling them to manage user information.
Private Routes for MEMBER:

MEMBER entities now have access to the "Activities" section
They can access the "Subscriptions" zone to review and manage their own subscriptions
Private Routes for SUPERADMIN:

SUPERADMIN entities have access  to manage ADMIN 

To test the logging of different entities you can use the following emails:
MEMBER:
firstmember1@gmail.com
ADMIN:
secondadmin@gmail.com
SUPER ADMIN:
superadmin2@gmail.com
the password for everyone is Abc123456

### CODE REVIEW

**Review by santimenendez98 — APPROVED** [review]

File: src/Components/Header/index.js [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
     }
-  }, [data]);
+  }, [token]);
+
+  const { show, message, state } = useSelector((state) => state.toast);
+  const history = useHistory();
   const handleLogout = () => {
-    localStorage.removeItem('login');
+    sessionStorage.removeItem('token');
+    sessionStorage.removeItem('role');
```
Comment (WalterArielPavicich) [inline src/Components/Header/index.js]: is redundant

File: src/Components/Header/index.js [hunk 2]
```diff
@@ -92,22 +74,22 @@ function Header() {
...
           )}
-          {layout.membership === 'classic' && (
+          {role === 'SUPER_ADMIN' && (
             <>
-              <Link to="/" className={styles.a}>
-                Home
+              <Link to="/admins" className={styles.a}>
+                Admins
+              </Link>
+              <Link to="/super-admins" className={styles.a}>
```
Comment (santimenendez98) [inline src/Components/Header/index.js]: This part is not necessary.

File: src/Routes/index.jsx [hunk 3]
```diff
@@ -37,34 +38,62 @@ const Layout = () => {
...
+              path="/super-admins/add"
+              role="SUPER_ADMIN"
+              component={SuperAdminsForm}
+            />
+            <PrivateRoute
+              exact
+              path="/super-admins/edit/:id"
+              role="SUPER_ADMIN"
+              component={SuperAdminsForm}
+            />
```
Comment (santimenendez98) [inline src/Routes/index.jsx]: This parts too

File: src/Components/Header/index.js [hunk 4]
```diff
@@ -92,22 +74,22 @@ function Header() {
...
-              <Link to="/super-admins" className={styles.a}>
-                Super Admins
-              </Link>
               <Link to="/trainers" className={styles.a}>
                 Trainers
               </Link>
             </>
           )}
-          {layout.membership === 'classic' && (
+          {role === 'SUPER_ADMIN' && (
```
Comment (santimenendez98) [inline src/Components/Header/index.js:80]: In this part maybe the source path should be added

**Review by santimenendez98 — CHANGES_REQUESTED** [review]

**Review by santimenendez98 — APPROVED** [review]

**Review by WalterArielPavicich — APPROVED** [review]
Great job!

**Comment by gabiholl** [comment]
There were some bugs reported in the next sections:
Admin classes https://trello.com/c/Eb5YGBzb 
Admin members https://trello.com/c/mMKQcCDf 

Superadmin: review bugs reported by QA partner Julián Riedinger
The rest of the itemps seem to work as expected.

**Pull request merged**